dev/com.ibm.ws.jpa_ol_fat/bnd.bnd

```diff
@@ -0,0 +1,34 @@
+#*******************************************************************************
+# Copyright (c) 2020 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+
+
+src: \
+	fat/src,\
+	test-applications/ConcurrentEnhancementVerification/src,\
+	test-applications/helpers/DatabaseManagement/src,\
```

Comment:
This source directory is missing from .classpath

Reply:
Ah, for DatabaseManagement?  Good catch, thanks!

Reply:
Fixed